config/optional/linkit.linkit_profile.az_linkit.yml

```diff
@@ -0,0 +1,44 @@
+langcode: en
```

Comment:
This can't be optional, since it is depended on by modules/custom/az_media/config/install/filter.format.az_standard.yml which isn't optional.

Reply:
I think the optional folder is for stand-alone config that nothing depends on, which is lame.